QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -18,6 +18,9 @@ public static void setupJoysticks() {
 		DriveCode.leftJoystickChannel = Igneous.joystick1.getYAxisSource();
 		DriveCode.rightJoystickChannel = Igneous.joystick1.getAxisSource(6);	
 		DriveCode.octocanumShifting = BooleanMixing.createDispatch(Igneous.joystick1.getButtonChannel(1), Igneous.globalPeriodic);
+		
+		Rollers.runRollersButton = BooleanMixing.createDispatch(Igneous.joystick2.getButtonChannel(3), Igneous.globalPeriodic);
+		Rollers.toggleRollersButton = BooleanMixing.createDispatch(Igneous.joystick2.getButtonChannel(4), Igneous.globalPeriodic);
```

Comment:
Since you're only going to be using the fact that these get pressed, consider just passing Events instead of Booleans. (`Igneous.joystick2.getButtonSource` will give you EventInputs directly.)